x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/statement.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { MouseEventHandler } from 'react';
+import { css } from '@emotion/react';
+import { EuiButtonEmpty, EuiCodeBlock, EuiFlexItem, UseEuiTheme } from '@elastic/eui';
+
+import { PluginStatement as PluginStatementModel } from '../models/pipeline/plugin_statement';
+import { CollapsibleStatement } from './collapsible_statement';
+import { IfElement } from '../models/list/if_element';
+import { PluginStatement } from './plugin_statement';
+import { Vertex } from './types';
+
+const spaceContainerStyle = (theme: UseEuiTheme) => css`
+  background-color: ${theme.euiTheme.colors.emptyShade};
+  align-self: stretch;
+  display: flex;
+  // Separates the left border spaces properly
+  border-bottom: solid 2px ${theme.euiTheme.colors.emptyShade};
+`;
+
+const spacerStyle = (theme: UseEuiTheme) => css`
+  width: ${theme.euiTheme.size.m};
+  align-self: stretch;
+  margin-left: ${theme.euiTheme.size.m};
+  border-left: 1px ${theme.euiTheme.border.color} dashed;
+
+  // This allows the border to be flush
+  &:last-child {
+    width: 0;
+  }
+
+  &:first-child {
+    // Odd number is because of the single pixel border
+    margin-left: calc(${theme.euiTheme.size.l}) - 1px);
+  }
+
+  @media (min-width: var(${theme.euiTheme.breakpoint.m})) {
+    border: none;
+  }
+`;
+
+const listItemStyle = (theme: UseEuiTheme) => css`
+  display: flex;
+  min-height: ${theme.euiTheme.size.xl};
+  align-items: center;
+  padding-right: ${theme.euiTheme.size.m};
+
+  &:nth-child(2n + 1) {
+    background: ${theme.euiTheme.colors.lightestShade};
+  }
+`;
+
+const conditionalStyle = css`
+  font-weight: bold;
```

Comment:
In theory, this could be `euiTheme.font.weight.bold` which would hook it up to the token definition of "bold" as opposed to the standard bold 700.